solr6/local/92-init-solr.sh

```diff
@@ -345,11 +345,28 @@ makeConfigs
 
 user="solr"
 # make sure backup folders exist and have the right permissions in case of mounts
-if [[ ! -d "${SOLR_DATA_ROOT}/solr6Backup" ]] || [[ $(stat -c %U "${SOLR_DATA_ROOT}/solr6Backup") != "$user" ]]
+if [ $SHARDING = true ]
 then
-    mkdir -p "${SOLR_DATA_ROOT}/solr6Backup/alfresco"  "${SOLR_DATA_ROOT}/solr6Backup/archive"
-	chown -hR "$user":"$user" "${SOLR_DATA_ROOT}/solr6Backup"
+  for i in $(echo $SHARD_IDS | tr "," "\n")
+  do
+	  solrCoreName=alfresco-$i
+	  mkdir -p "${SOLR_DATA_ROOT}/solr6Backup/$solrCoreName"
```

Comment:
We are not using `DIR_ROOT`here?

Reply:
In the case of solr1 and solr4, both alfresco and solr and installed in /opt/alfresco - so dir.root (DIR_ROOT) has a meaning.
In the case of solr6, it is installed in /opt/alfresco-search-services (folder included in the zip file of the distribution), therefore dir.root from alfresco does not have any meaning.

Reply:
Okay, the default location is changed. But here it is forced to be within the same directory used for indexes, and you can't change that.
In alfresco, you can still overwrite the global property to change the backup location if you want to, but here you are stuck.